.rhcicd/frontend.yml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: api-frontend
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: devprod
+    spec:
+      envName: ${ENV_NAME}
+      title: API Frontend
+      deploymentRepo: https://github.com/RedHatInsights/api-frontend
+      API:
+        versions:
+          - v1
```

Comment:
The API field is not needed here, since it doesn't have any API associated to it.
```suggestion
```